src/components/__tests__/Button.spec.tsx

```diff
@@ -1,88 +1,93 @@
+import React, { ComponentProps } from 'react'
 import { Text } from 'react-native'
-import React from 'react'
-import Button, { styles, IButtonProps } from '../Button'
-import { shallow } from 'enzyme'
-import toJson from 'enzyme-to-json'
+import Button from '../Button'
+import { shallow, ShallowWrapper } from 'enzyme'
 import { BUTTONS_THEME } from '../../constants/data'
+import theme from '../../constants/theme'
+import { COLORS } from '../../constants/theme/colors'
+import { mocked } from 'ts-jest/utils'
+import wrapWithTheme from '../../testing/wrapWithTheme'
+
+type ButtonPropsType = ComponentProps<typeof Button>
+
+const expectStylesToMatch = (component: ShallowWrapper<ButtonPropsType>, expectedStyles: Record<string, any>): void => {
+  expect(component.dive().dive().dive().prop<Record<string, any>>('style')[0]).toEqual(
```

Comment:
Not really pretty. Apparently there is a [package for testing styled components ](https://github.com/styled-components/jest-styled-components). But it is currently not working for native. I'll keep my eyes open on this.